app/assets/javascripts/discourse/app/controllers/user-badges.js

```diff
@@ -1,14 +1,25 @@
 import Controller, { inject as controller } from "@ember/controller";
-import { alias, sort } from "@ember/object/computed";
+import { action, computed } from "@ember/object";
+import { alias, filterBy, sort } from "@ember/object/computed";
 
 export default Controller.extend({
   user: controller(),
   username: alias("user.model.username_lower"),
   sortedBadges: sort("model", "badgeSortOrder"),
+  favoriteBadges: filterBy("model", "is_favorite", true),
+  favoriteCount: alias("favoriteBadges.length"),
```

Comment:
alias is probably not needed for this line and next one

Reply:
Fair enough, I am still learning the fundamentals of Ember and I'm not sure when it is good practice to use certain things. Definitely good to get your input!